README.md

```diff
@@ -1,2 +1,28 @@
 # Hello-Microverse-Project
 Day 2 Week 1 Project Of Setting Up "Hello Mivroverse" With HTML And CSS Project
+
+# Built with
+<ul>
+<li>HTML</li>
+<li>CSS</li>
+</ul>
+
+# Getting Started
+The following is the instructions for creating this project. Follow to understand.
+
+# Prerequisites
+VSCode
+Git
+Browser
+ # Setup
+<p>Create a directory in Github.<br>
+Clone the directory to local repository and make it available locally.<br>
+From the main branch,create a branch eg:  file-creation branch  to create the necessary files required for the project which will be merged to the main branch. These Files include index.html and style.css.<br>
+Then create-linter-files branch  to create the linter files. Then later will be merged to the main branch.<br>
+Open VSCode, choose open folder and open the directory that was cloned from Github.<br>
+Open the index.html and populate it with boiler plate code and a h1 element with the text Hello Microverse and assign a class to it of name header
+Link the stylesheet to index.html.<br>
+Open the style.css and add the style properties to the h1 element.
+
+# Run Tests
+Run the index.html on your preferred browser or use a live server
```

Comment:
- [ ] You have added a readme file great :+1: but I think you forgot some headings and their content. Please compare your readme with this [repo readme](https://github.com/HAMMAS-SALEEM/Hello-Microverse) and kindly add the missing headings and their content :thinking: